pvlib/pvsystem.py

```diff
@@ -2537,6 +2553,73 @@ def pvwatts_losses(soiling=2, shading=3, snow=0, mismatch=2, wiring=2,
     return losses
 
 
+def irradiance_loss_pvsyst(effective_irradiance, irradiance_shading_loss,
+                           irradiance_snow_loss, irradiance_soiling_loss):
+    r"""
+    Calculates plane of array irradiance losses using a model based on the
+    PVSyst methodology [1]_ of linearly reducing the effective irradiance on a
+    module.  This doesn't account for the electrical losses due to irregular
+    irradiance on different solar cell strings but rather reduces the
+    irradiance input to the single diode equation.
+
+    The losses are compounded using the following equation:
+
+    .. math::
+
+        L_{total}(\%) = 100 [ 1 - \Pi_i ( 1 - \frac{L_i}{100} ) ]
+
```

Comment:
:math:`L_{total}` is the total irradiance loss resulting from the effects of shading, snow and soiling.

Reply:
I am suggesting a new line for the docstring here.